code/game/machinery/computer/prisoner.dm

```diff
@@ -10,7 +10,7 @@
 	var/status = 0
 	var/timeleft = 60
 	var/stop = 0.0
-	var/screen = 0 // 0 - No Access Denied, 1 - Access allowed
+	var/screen = FALSE // FALSE - No Access Denied, TRUE - Access allowed
```

Comment:
I don't think moving this to TRUE and FALSE really makes this code more readable by itself. Could you rename this variable to `authenticated` or something similar? That would much better reflect what this variable is actually doing and better justify TRUE/FALSE values. In its current state, a variable named `screen` moreso indicates that the variable is storing the current screen state.

Reply:
Done